example/notify_inval_entry.c

```diff
@@ -258,8 +268,22 @@ static void* update_fs_loop(void *data) {
         update_fs();
         if (!options.no_notify && lookup_cnt) {
             if(options.only_expire) {
-                assert(fuse_lowlevel_notify_expire_entry
-                   (se, FUSE_ROOT_ID, old_name, strlen(old_name), FUSE_LL_EXPIRE_ONLY) == 0);
+                int ret = fuse_lowlevel_notify_expire_entry
+                   (se, FUSE_ROOT_ID, old_name, strlen(old_name));
+                if (ret == -ENOSYS) {
+                    printf("fuse_lowlevel_notify_expire_entry not supported by kernel\n");
+                    printf("Next call to mountpoint will kill it...\n");
+                    raise(SIGINT);
```

Comment:
That seems somewhat over complicated to me. Let's either print an error message and continue, or print an error and terminate the session right away (with fuse_session_exit()).

Reply:
sorry my mistake.. for whatever reason i overlooked that the `fuse_session` is given as parameter in this function